app/concerns/liquid_droppable.rb

```diff
@@ -82,9 +84,49 @@ def to_liquid
     end
   end
 
+  class ActiveRecordCollectionDrop < Drop
+    include Enumerable
+
+    def each(&block)
```

Comment:
Can't we `delegate` the methods we want to support (or every call) to the `CollectionProxy`?

Reply:
It could work, but generally, Liquid cares about security and prefers explicit method definitions, so I avoided depending on ActiveSupport features for now.  Maybe I'll do that later along with other Drop classes because most of them involve simple delegation. 😊

Reply:
I understand that we don't want to delegate all calls, but `each`, `fetch`, `first` and `last` could be`def_delegators` from `Forwardable`, right?